packages/core/state/interaction/logics.ts

```diff
@@ -70,20 +70,31 @@ const initializeApp = createLogic({
         const isOnWeb = interactionSelectors.isOnWeb(deps.getState());
         const fileService = interactionSelectors.getHttpFileService(deps.getState());
 
-        // Redimentary check to see if the user is an AICS Employee by
+        // Rudimentary check to see if the user is an AICS employee by
         // checking if the AICS network is accessible
         const isAicsEmployee = await fileService.isNetworkAccessible();
 
-        // If there are query args representing a query we can extract that
-        // into the query to render (ex. when refreshing a page)
+        // If the url has search params representing a query, we can extract those
+        // into a query object to render (ex. when refreshing a page)
         if (isOnWeb && window.location.search) {
             try {
-                dispatch(
-                    selection.actions.addQuery({
-                        name: DEFAULT_QUERY_NAME,
-                        parts: SearchParams.decode(window.location.search),
-                    })
-                );
+                // If the exact query already exists in state on initialization,
+                // we shouldn't add it again-- just refresh the existing query
+                if (
+                    queries.some(
+                        (query) =>
+                            `?${SearchParams.encode(query?.parts)}` === window.location.search
+                    )
+                ) {
+                    dispatch(refresh);
```

Comment:
Why do we need the refresh?

Reply:
It was a precautionary measure in case the data had become out of date and/or the correct query results weren't being rendered, but may not be necessary -- will test